bats/tests/helpers/commands.bash

```diff
@@ -49,8 +49,10 @@ docker() {
     docker_exe --context $RD_DOCKER_CONTEXT "$@"
 }
 docker_exe() {
-    # Add path to bundled credential helpers to the front of the PATH
```

Comment:
Funny that we already had the code; it was just not working properly on Windows. 😄